lib/msal-core/src/AuthResponse.ts

```diff
@@ -4,14 +4,98 @@ import { IdToken } from "./IdToken";
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export type AuthResponse = {
-    uniqueId: string;
-    tenantId: string;
-    tokenType: string;
-    idToken: IdToken;
-    accessToken: object;
-    scopes: Array<string>;
-    expiresIn: string;
-    account: User;
-    userState: string;
-};
+export class AuthResponse  {
+    private uniqueId: string;
+    private tenantId: string;
+    private tokenType: string;
+    private idToken: IdToken;
+    private accessToken: object;
+    private scopes: Array<string>;
+    private expiresIn: string;
```

Comment:
what is an example value here?

Reply:
This comes from the server:
&expires_in=3600

We can make this a number instead of string, but I figured for simplicity we just pass back as a string.

Reply:
if we are just returning the number, lets return the number as a number,
Let's developers create time check on their own end if they want

Reply:
+1. Eventually we should be returning a Date imo

Reply:
agreed, probably want server to return the date, so we aren't trying to guess now + milliseconds